src/unix/linux_like/linux/mod.rs

```diff
@@ -2979,6 +2979,19 @@ pub const MSG_NOERROR: c_int = 0o10000;
 pub const MSG_EXCEPT: c_int = 0o20000;
 pub const MSG_ZEROCOPY: c_int = 0x4000000;
 
+pub const SEM_UNDO: c_int = 4096;
```

Comment:
Could you change the literal to 0x1000 to match the Linux header? Please squash, otherwise lgtm